components/skin/ui/src/main/resources/PhenoTips/outdatedBrowser.xml

```diff
@@ -0,0 +1,297 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<xwikidoc version="1.1">
+  <web>other</web>
+  <name>outdatedBrowser</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>XWiki.Admin</creator>
+  <creationDate>1438054658000</creationDate>
+  <author>XWiki.Admin</author>
```

Comment:
Add the following line between the `creationDate` and `author`:

    <parent>PhenoTips.Skin</parent>